ramp-engine/ramp_engine/aws/api.py

```diff
@@ -251,26 +251,28 @@ def launch_ec2_instances(config, nb=1):
 def _get_image_id(config, image_name):
     sess = _get_boto_session(config)
     client = sess.client('ec2')
+
+    # get all the images with the given imange_name in the name
     result = client.describe_images(Filters=[
         {
             'Name': 'name',
-            'Values': [
-                image_name
-            ]
+            'Values': [image_name+'*'
+                       ],
```

Comment:
This is quite efficient :+1: 
Just a small nitpick
```suggestion
            'Values': [
                f'{image_name}*'
            ],
```